pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -129,31 +139,20 @@ func (c *Cluster) Start() error {
 	}
 
 	logger.Infof("start running mgr")
-
-	for i := 0; i < c.Replicas; i++ {
-		if i >= 2 {
-			logger.Errorf("cannot have more than 2 mgrs")
```

Comment:
Why are we suddenly supporting more than 2 mgrs? Also, I thought we only supported 1 mgr ... is that verfied in the CRD spec?

Reply:
`Replicas` is currently hard-coded to 1, it's not configurable. But I meant to move this check into `getDaemonIDs()`, thanks for catching it.